misc/windows-deploy/user-data.ps1

```diff
@@ -0,0 +1,52 @@
+<powershell>
+## The string 'windows' shoud be replaced with 
+## template variable for cluster name.
+
+# Set agent env variables for the Machine context (durrable)
+[Environment]::SetEnvironmentVariable("ECS_CLUSTER", "windows", "Machine")
+[Environment]::SetEnvironmentVariable("ECS_CRED_PROXY_ENABLED", "False", "Machine")
+$agentVersion = '1.14.0-1.windows.1'
+$agentZipHash = '7748b1d3c73d5211e150db746cdec5bf'
+$agentZipUri = "https://s3-us-west-2.amazonaws.com/windows-agent/ecs-agent-windows-$agentVersion.zip"
```

Comment:
It's not published yet, but the URL will end up being `https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-windows-${agentVersion}.zip`.  This is the same bucket and almost the same naming scheme as Linux (the only differences are `-windows` and `.zip` instead of `.tar`).